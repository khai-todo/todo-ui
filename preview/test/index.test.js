import Subject from '../pages/index'
import renderer from 'react-test-renderer'

it('render correctly', () => {
  expect(
    renderer.create(<Subject />).toJSON()
  ).toMatchSnapshot()
})
