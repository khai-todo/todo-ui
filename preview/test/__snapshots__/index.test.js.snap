// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<nav>
  <ul>
    <li>
      <a
        href="/editor"
        onClick={[Function]}
      >
        Editor
      </a>
    </li>
    <li>
      <a
        href="/viewer"
        onClick={[Function]}
      >
        Viewer
      </a>
    </li>
  </ul>
</nav>
`;
